dynamodb/client.go

```diff
@@ -0,0 +1,99 @@
+package dynamodb
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+)
+
+const (
+	TableHashKey = "conceptId"
+)
+
+type ConcordancesModel struct {
+	UUID         string   `json:"conceptId"`
+	ConcordedIds []string `json:"concordedIds"`
+}
+
+type Client interface {
+	Read(uuid string) (ConcordancesModel, error)
+	Write(m ConcordancesModel) (ConcordancesModel, error)
+	Delete(uuid string) (ConcordancesModel, error)
+}
+
+type DynamoDBClient struct {
+	dynamoDbTable string
+	awsRegion     string
+	ddb           dynamodbiface.DynamoDBAPI
```

Comment:
Your tests use the direct implementation rather than a mock, so I personally wouldn't have used the interface here.